apps/taquito-test-dapp/src/tests.ts

```diff
@@ -560,6 +562,8 @@ export const init = (
     id: "contract-call-simple-type",
     name: "Contract call with int",
     description: "This test calls a contract entrypoint and passes an int",
+    documentation: 'https://tezostaquito.io/docs/quick_start#interact-with-a-smart-contract',
```

Comment:
I think this link would be more appropriate: https://tezostaquito.io/docs/smartcontracts